lumigator/python/mzai/backend/settings.py

```diff
@@ -88,7 +88,7 @@ def RAY_WORKER_GPUS_FRACTION(self) -> float:  # noqa: N802
     def PIP_REQS(self) -> list:  # noqa: N802
         return [
             line.strip()
-            for line in Path.open("lumigator/python/mzai/lm_buddy/requirements.txt")
+            for line in Path.open("lumigator/python/mzai/evaluator/requirements.txt")
```

Comment:
see comment above

Reply:
We'll still need to eventually point to the requirements path in the evaluator directory, unless I'm misunderstanding